solidus_support.gemspec

```diff
@@ -28,6 +28,9 @@ Gem::Specification.new do |spec|
   spec.executables = files.grep(%r{^exe/}) { |f| File.basename(f) }
   spec.require_paths = ["lib"]
 
+  spec.add_dependency 'flickwerk', '~> 0.2.0'
+  spec.add_dependency 'solidus_core', '~> 4.1'
```

Comment:
Can you explain how this change is related to your PR?

Reply:
Yes. For any patches to `Spree.user_class`, Flickwerk needs `Spree.user_class_name` to be present, and that method is only available from Solidus 4.1 upwards.